src/utils/exportUtils/exportJS.ts

```diff
@@ -0,0 +1,27 @@
+export default `
+function showToastIfNeeded(replyId){
+  let el = document.getElementById(replyId);
+  if(!el) { 
+      showToast("The message you're looking for wasn't exported");
+      return;
+  };
+}
+
+function showToast(text) {
+  let el = document.getElementById("snackbar");
+  el.innerHTML = text;
+  el.className = "mx_show";
+  setTimeout(() => {
+      el.className = el.className.replace("mx_show", "");
+  }, 2000);
+}
+
+window.onload = () => {
+document.querySelectorAll('.mx_reply_anchor').forEach(element => {
+  element.addEventListener('click', event => {
+    showToastIfNeeded(event.target.getAttribute("scroll-to"));
+  })
+})
```

Comment:
So right now `.mx_reply_anchor`s need this JS to work, it'd be nice if it could be done using anchors, and if the message wasn't exported then maybe opening a matrix.to tab instead or similar as I can imagine quite a few people would like to see this work with NoJS

Reply:
If you're referring to jumping to replies, it is done using anchors. JS is used only to show toasts for non exported events. I will configure it to link to the original message using matrix.to. But finding if an event is exported for every reply event and linking it appropriately can be quite expensive, I think. If it's fine with you, I'll do that instead.